lib/sqlalchemy/dialects/sqlite/base.py

```diff
@@ -273,8 +273,9 @@ def bi_c(element, compiler, **kw):
 .. warning::
 
     SQLite's SAVEPOINT feature is impacted by unresolved
-    issues in the pysqlite driver, which defers BEGIN statements to a greater
-    degree than is often feasible. See the section :ref:`pysqlite_serializable`
+    issues in the pysqlite and aiosqlite driver, which defers BEGIN statements
```

Comment:
**mike bayer** (zzzeek) wrote:

"drivers"

"defer"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907